### DISCUSSION TITLE
IPD distance of around 6.5 cm - Any thoughts on settings

### DISCUSSION THREAD

**Discussion by gateway** [thygate/stable-diffusion-webui-depthmap-script#351] [Q&A]
Hi, I'm working on converting some of my stuff for Stereoscopy 3D, I need to create the IPD setting of around 6.5 cm if it was real world camera settings.. any thoughts.. Also Ill need to output it as top and bottom which left eye is on the top and right on the bottom.. I notice their are settings for top-bottom, bottom-top, but what eye is on top :)

Any thoughts on some settings to try, their is divergence which by default is 2.5.. 2.5 what pixels apart? What about separation in After effects I usually do 11+ pixels for the left and -11 for the right would this be similar or is this the balance between eyes in pixels? 

Sorry for all the questions, I'm used to using AE or Blender to settings in the cameras im using.

I'm trying to recreate this as close as possible

![image](https://github.com/thygate/stable-diffusion-webui-depthmap-script/assets/519563/16a4c12f-0fa6-462f-8920-3be0cbb8ff90)

![image](https://github.com/thygate/stable-diffusion-webui-depthmap-script/assets/519563/8cb21f35-9456-4424-823e-67c12490bc10)

Would love to hear your thoughts

**Comment by sonakravtsenko3** [comment]
Hi! Top-bottom mode only outputs image where left eye image is on top, the bottom image is for the right eye. Both divergence and separation are measured in percentages of width of the original image. This is not the most scientific approach, I would love somebody to add an option to specify physical characteristics instead of the percentages.

**Reply by gateway** [reply]
Thanks for your reply, its been really hard to find that sweet spot, so for now im taking the depth map and adding doing the work in after effects where i can set a displacement on the the main footage with the depth as the displacement, then i set it to -11 for left and then duplicate it for the right for +11 which gives me about 22 pixels , then I apply the 3d Glasses effect and render out.. but would love to figure out a more automatic way to deal wit pixels and inches etc.. thank you again for all the hard work.
